docs/Actions/run-sql.md

```diff
@@ -0,0 +1,52 @@
+---
+id: run-sql
+title: Run SQL Query
+sidebar_label: Run SQL Query
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `SQL`
+
+A simple action which allows to query the database for data.
+
+## `Typical Use Cases`
+
+- Query the database for specific data
+- Insert data into tables
+- Pre-load data into Action Form
+
+## `Don't use it to`
+
+- Load multiple sets of data - use [Load Entities (SQL)](Actions/load-entities-sql.md) instead
+
```

Comment:
add: query entities; prefer using the entity related actions when possible.
add: update system tables directly, it might break functionality. use specific actions instead.